### PR TITLE
manual suricata

### PR DESCRIPTION
**Pull request by mkorobeinikov**
manual suricata vyos 1.5

### CODE REVIEW

File: docs/configuration/service/suricata.rst [hunk 1]
```diff
@@ -0,0 +1,105 @@
+.. _suricata:
+
+########
+suricata
+########
+
+:abbr:`suricata` Suricata and VyOS are powerful tools for ensuring network security and traffic management.
```
Comment (rebortg) [inline docs/configuration/service/suricata.rst]: the :abbr: does not make sense here, it is for abbreviation and the long form, please remove it.

File: docs/configuration/service/suricata.rst [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+########
+
+:abbr:`suricata` Suricata and VyOS are powerful tools for ensuring network security and traffic management. 
+Suricata is an open-source intrusion detection and prevention system (IDS/IPS) that analyzes network packets in real-time. 
+
+
+Suricata Features
+=================
+
+Intrusion Detection (IDS): Analyzes network traffic and detects suspicious activities, attacks, and malicious traffic.
```
Comment (rebortg) [inline docs/configuration/service/suricata.rst:14]: this points looks odd in the rendered page
look [here](https://vyos--1509.org.readthedocs.build/en/1509/configuration/service/suricata.html).

maybe bullet point will fit better?

File: docs/configuration/service/suricata.rst [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+   +> address-group        Address group name
+   +  interface            Interface to use
+    > log                  Suricata log outputs
+   +> port-group           Port group name
+
+These commands create a flexible interface for configuring the Suricata service, allowing users to specify addresses, ports, 
+and logging parameters.
+
+After completing the service configuration in configuration mode, the main configuration file suricata.yaml is created, 
+into which all specified parameters are added. Then, to ensure proper operation, the command $ update suricata must be run
```
Comment (rebortg) [inline docs/configuration/service/suricata.rst]: please use the inline opcmd
https://docs.vyos.io/en/sagitta/documentation.html#opcmd

**Review by rebortg — CHANGES_REQUESTED** [review]

File: docs/configuration/service/suricata.rst (hunk 2 above)
Comment (mkorobeinikov) [reply docs/configuration/service/suricata.rst:14]: I just removed the indents between the lines, it made it better.

**Comment by mkorobeinikov** [comment]
Corrections have been made.

**Pull request merged**